src/components/application_manager/test/commands/mobile/create_interaction_choice_set_test.cc

```diff
@@ -213,11 +212,10 @@ TEST_F(CreateInteractionChoiceSetRequestTest, OnEvent_VR_UNSUPPORTED_RESOURCE) {
   utils::SharedPtr<CreateInteractionChoiceSetRequest> req_vr =
       CreateCommand<CreateInteractionChoiceSetRequest>(msg_vr);
 
-  MockAppPtr mock_app = CreateMockApp();
-  ON_CALL(app_mngr_, application(_)).WillByDefault(Return(mock_app));
-  ON_CALL(*mock_app, app_id()).WillByDefault(Return(kConnectionKey));
+  ON_CALL(app_mngr_, application(_)).WillByDefault(Return(app_));
+
   smart_objects::SmartObject* null_obj = NULL;
-  ON_CALL(*mock_app, FindChoiceSet(_)).WillByDefault(Return(null_obj));
+  ON_CALL(*app_, FindChoiceSet(_)).WillByDefault(Return(null_obj));
```

Comment:
@dev-gh I would recommend to rename this class member to `mock_app_` according to [Unit testing in Project](https://adc.luxoft.com/confluence/pages/viewpage.action?pageId=267764838)